Cargo.lock

```diff
@@ -1526,8 +1526,6 @@ dependencies = [
  "deno_fs",
  "deno_media_type",
  "deno_net",
- "deno_npm",
- "deno_semver 0.5.1",
```

Comment:
This 0.5.1 is not ideal. I opened https://github.com/denoland/deno_config/pull/13